### PR TITLE
Build

### PR DESCRIPTION
**Pull request by jaanonim**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +16,7 @@
         "type": "git",
         "url": "git+https://github.com/n2oneProgrammer/atari-logo-interpreter.git"
     },
-    "author": "",
+    "author": "jaanonim",
```
Comment (n2oneProgrammer) [inline package.json]: expedition to mordor

**Review by n2oneProgrammer — APPROVED** [review]
LGTM

File: package.json (hunk 1 above)
Comment (jaanonim) [reply package.json]: ;-)

**Review by Adron4444 — APPROVED** [review]
LGTM

**Pull request merged**